internal/client/user.go

```diff
@@ -44,3 +46,31 @@ func (c *Client) GetCurrentUser() (models.User, error) {
 
 	return u, nil
 }
+
+// PatchAddUserToZone patches a user into a zone
+func (c *Client) PatchAddUserToZone(zoneID string) (models.PatchUser, error) {
```

Comment:
```suggestion
// MoveCurrentUserToZone moves the current user into a given zone
func (c *Client) MoveCurrentUserToZone(zoneID uuid.UUID) (models.User, error) {
```

Returns a `User`. https://github.com/redhat-et/apex/blob/main/internal/handlers/user.go#L47